rustls-test/src/lib.rs

```diff
@@ -2095,11 +2095,13 @@ pub mod encoding {
         pub fn new_sig_algs() -> Self {
             Self {
                 typ: ExtensionType::SignatureAlgorithms,
-                body: len_u16(
-                    SignatureScheme::RSA_PKCS1_SHA256
-                        .to_array()
-                        .to_vec(),
-                ),
+                body: len_u16(vector_of(
```

Comment:
`RSA_PKCS1_SHA256` is not allowed in TLS 1.3, and this will now yield earlier errors in tests that depend on 1.3 (in this case, QUIC and the failure test of no transport parameters).